podpac/core/data/datasource.py

```diff
@@ -338,27 +340,30 @@ def _eval(self, coordinates, output=None, _selector=None):
         ]
         coordinates = coordinates.drop(extra)
 
-        # store input coordinates to evaluated coordinates
-        self._evaluated_coordinates = deepcopy(coordinates)
```

Comment:
I believe that this deepcopy was important...

Reply:
Yeah, the coordinates are changed after this because the crs is transformed. But I think the only time the deepcopied self._evaluated_coordinates is used is for it's dims, which don't change. I'll make that more clear.

Reply:
I'm not sure it's better, but I clarified that in the subsequent commit.